src/launcher/java/org/truffleruby/launcher/options/CommandLineOptions.java

```diff
@@ -54,6 +53,10 @@
     private boolean forceStdin;
     private boolean shouldPrintShortUsage;
 
+    // TODO (pitr-ch 26-Jul-2017): Move to Options when implementing -s option
+    // Currently not used
```

Comment:
I would like to implement this on our side soon, since we already have the parsing in `processArgv` method. Hm I could delete it and keep a hash of a commit to revert around in the comment though.